drivers/modem/ublox-sara-r4.c

```diff
@@ -876,11 +1090,37 @@ static void modem_reset(void)
 		goto restart;
 	}
 
-	ret = modem_cmd_handler_setup_cmds(&mctx.iface, &mctx.cmd_handler,
+#if defined(CONFIG_MODEM_UBLOX_SARA_AUTODETECT_VARIANT)
+	if (mdata.mdm_variant == MDM_VARIANT_UBLOX_U2) {
+
+#if defined(CONFIG_MODEM_UBLOX_SARA_AUTODETECT_APN)
+		/* setup PDP context definition */
+		char cmd[64];
```

Comment:
Similar thing here

Reply:
Thank you for your input. I will look at this within a few days. It it ok for the workflow if I add commits, or should I squash them again, replacing the original commit?

Reply:
As a general rule, if you are fixing things in your own commits, then squash. If you are introducing new logically separate stuff, then do new commit. In this case, just squash the commits.